03/app.js

```diff
@@ -1,42 +1,74 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
 
 class Article extends React.Component {
-    state = {
-        comments: [],
-    }
-    
-    render() {
-        const {title, body} = this.props;
-        return (
-            <article>
-                <h1>{ title }</h1>
-                <p>{ body }</p>
-                <section>
-                    <form>
-                        <div>
-                            <label>
-                                <textarea 
-                                    style={{ "min-width": "300px", "min-height": "120px" }} 
-                                    name="content" 
-                                />
-                            </label>
-                        </div>
-                        <div><input type="submit" value="dodaj komentarz" /></div>
-                    </form>
-                    <ul>
-                        {/* tutaj komentarze jako <li/>, ps. tak wygląda komentarz do kodu w JSX */}
-                    </ul>
-                </section>
-            </article>
-        )
+  state = {
+    textarea: "",
+    comments: [],
+  };
+
+  render() {
+    const { title, body } = this.props;
+    const { textarea } = this.state;
+    return (
+      <article>
+        <h1>{title}</h1>
+        <p>{body}</p>
+        <section>
+          <form onSubmit={this.submitHandler}>
+            <div>
+              <label>
+                <textarea
+                  onChange={this.changeHandler}
+                  style={{ minWidth: "300px", minHeight: "120px" }}
+                  name="content"
+                  value={textarea}
+                />
+              </label>
+            </div>
+            <div>
+              <input type="submit" value="dodaj komentarz" />
+            </div>
+          </form>
+          <ul>{this.renderCommentsList()}</ul>
+        </section>
+      </article>
+    );
+  }
+
+  changeHandler = (e) => {
+    this.setState({
+      textarea: e.target.value,
+    });
+  };
+
+  submitHandler = (e) => {
+    e.preventDefault();
+    const { textarea } = this.state;
+    if (textarea) {
+      this.addComment(textarea);
+      this.setState({ textarea: "" });
+    } else {
+      alert("blank comment");
     }
+  };
+
+  addComment(textarea) {
```

Comment:
Raczej parametr nazwałbym przez `content` lub `text`. Twoja nazwa nie koniecznie jest dobra to może się okazać, że ta metoda przyjmie też dane z innego elementu niż `textarea`.

Reply:
Okej, rozumiem